cmd/cli/namespace_add.go

```diff
@@ -16,11 +16,22 @@ import (
 )
 
 const namespaceAddDescription = `
-This command will add a namespace or a set of namespaces
-to the mesh. Optionally, the namespaces can be configured
-for automatic sidecar injection which enables pods in the
+This command will add a namespace or a set of namespaces 
+to the mesh, automatic sidecar injection is disabled by 
+default. Optionally, the namespaces can be configured for 
+automatic sidecar injection which enables pods in the 
```

Comment:
```suggestion
This command will add a namespace or a set of namespaces 
to the mesh so that osm-controller can observe resources belonging
to mesh namespaces, automatic sidecar injection is disabled by 
default. Optionally, the namespaces can be configured for 
automatic sidecar injection which enables pods in the 
```

@patricekrakow does this make it more clear?

Reply:
Yes, it does.